src/renderer/dx/DxFontRenderData.cpp

```diff
@@ -0,0 +1,758 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+#include "precomp.h"
+
+#include "DxFontRenderData.h"
+
+static constexpr float POINTS_PER_INCH = 72.0f;
+static constexpr std::wstring_view FALLBACK_FONT_FACES[] = { L"Consolas", L"Lucida Console", L"Courier New" };
+static constexpr std::wstring_view FALLBACK_LOCALE = L"en-us";
+
+using namespace Microsoft::Console::Render;
+
+DxFontRenderData::DxFontRenderData(::Microsoft::WRL::ComPtr<IDWriteFactory1> dwriteFactory) noexcept :

```

Comment:
I don't know how I should feel about this single file being over 700 LOC 😅 This is some serious refactoring shiz, man.

Reply:
I mean.... `DxRenderer.cpp` is a monster. The fact that this is ONLY 700 lines is better... Take what small wins we can get.